src/python/pants/build_graph/target.py

```diff
@@ -658,14 +663,18 @@ def compute_injectable_specs(cls, kwargs=None, payload=None):
     :param Payload payload: The post-Target.__init__() Payload object.
     :yields: AddressSpec strings representing dependencies of this target.
     """
+    cls._compute_injectable_specs_warn_or_error()
+    cls.compute_injectable_address_specs(kwargs=kwargs, payload=payload)
+
+  @classmethod
+  def _compute_injectable_specs_warn_or_error(cls):
     warn_or_error(
       removal_version="1.27.0.dev0",
       deprecated_entity_description="Target.compute_injectable_specs()",
       hint="Use `Target.compute_injectable_address_specs()` instead. The API is the same as "
            "before. This change is to prepare for Pants eventually supporting file system specs, "
            "e.g. `./pants cloc foo.py`. In preparation, we renamed `Spec` to `AddressSpec`."
```

Comment:
Consider modernizing:

```
"Use `Target.compute_injectable_address_specs()` instead. The API is the same as "
"before. This change is because Pants now supports file args, e.g. "
"`./pants cloc foo.py`. In preparation, we renamed `Spec` to `AddressSpec`."
```